lib/player/human.rb

```diff
@@ -0,0 +1,10 @@
+module Players
+  
+class Human < Player
+  attr_accessor :board
+  
+  def move(board)
+    input=gets.strip

```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```